tests/testthat/test-exporters.R

```diff
@@ -431,3 +431,16 @@ test_that("export_as_doc works thanks to tt_to_flextable", {
 
   expect_true(file.exists(doc_file))
 })
+
+test_that("export_as_doc works thanks to tt_to_flextable", {
```

Comment:
`flextable` is in suggests then it would be good to have skip_if_not_installed clause somewhere there to protect against so-called noSuggest check

Reply:
done